assets/scss/components/_announcement-banner.scss

```diff
@@ -0,0 +1,34 @@
+.c-announcement-banner {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: $color-black;
+    padding: 1rem 1.5rem;
+    position: relative;
+    z-index: $z-index-mid;
+
+    p {
```

Comment:
is the announcement bar as Wysiwyg container? If not, could we have a dedicated class names for the nested elements?